stdlib/pagerduty/pagerduty.flux

```diff
@@ -143,7 +149,16 @@ sendEvent = (
         "Accept": "application/vnd.pagerduty+json;version=2",
         "Content-Type": "application/json",
     }
-    enc = json.encode(v: data)
+    enc = if customDetails == record.any then
```

Comment:
why do we need this conditional? Is there an error encoding an empty JSON object as the custom details?